app/src/main/java/com/willowtree/vocable/BaseActivity.kt

```diff
@@ -40,7 +40,11 @@ abstract class BaseActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        if (BuildConfig.USE_HEAD_TRACKING && !checkIsSupportedDeviceOrFinish()) {
+
+        val shouldForceDisableHeadTracking = !BuildConfig.USE_HEAD_TRACKING
+        val isSupportedDevice = checkIsSupportedDeviceOrFinish()
+
+        if (shouldForceDisableHeadTracking || !isSupportedDevice) {
```

Comment:
I'm not sure, but this check might also be in the `MainActivity` and the `SettingsActivity`